### DISCUSSION TITLE
Delivered to internal network by a host with no rDNS

### DISCUSSION THREAD

**Discussion by Toni202012569** [postalserver/postal#2700] [Help with using Postal]
Good evening.

I have a problem with Postal. It stopped giving out the rDNS record. And as a result, problems began with the delivery of letters. I contacted the hoster about this issue, but they cannot explain anything to me. If I connect other servers, there are no problems. It is obvious that there is some kind of problem with Postal. But I can’t figure out how to solve it.

Delivered to internal network by a host with no rDNS
This may indicate you do not have a rDNS configured for your hostname or the rDNS does not match your sending IP

**Comment by willpower232** [comment]
If you have a go with https://www.mail-tester.com/ then that usually points out any issues like this.

If you have a problem with reverse DNS then you may need to consult the documentation for your server provider as they will be the ones responsible for adding it as they manage the IP addresses but probably provide you some kind of interface.
